build.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# ****************************************************************************
+# Copyright 2004-2013 Castle Project - http://www.castleproject.org/
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ****************************************************************************
+
+clear
```

Comment:
I prefer this, between runs it helps make the latest output easier to read. It also does not really clear your console because if you scroll up hard enough you will still find your buffered output in bash. It also appears to have no effect in TravisCI either.